R/corpus-methods-quanteda.R

```diff
@@ -208,7 +208,11 @@ docnames <- function(x) {
 docnames.corpus <- function(x) {
     # didn't use accessor documents() because didn't want to pass
     # that large object
-    rownames(x$documents)
+    if (is.null(rownames(x$documents))) {
```

Comment:
This seems slightly dangerous, since it means we could have a corpus object with no document names, but that `docnames()` will make it seem like it does have document names. Have you tested all of the functions that rely on there being actual document names? If they all call `docnames.corpus()` then we should be OK, since this will create the names.